keyboards/adpenrose/akemipad/keymaps/oled_bongocat/keymap.c

```diff
@@ -0,0 +1,202 @@
+    } else if (via_layouts.split_zero && !(via_layouts.split_enter) && !(via_layouts.split_plus)){
+      xprintf("SZ is the only layout active with a value of: %d\n", value); 
+    } else {
+      xprintf("The base layout is active with a value of: %d\n", value);
+    }
+  #endif
+}
+
+/* Turing LEDs on/off depending on the layout selected on VIA */
+void rgb_matrix_indicators_user(void){
+  if (via_layouts.split_zero && via_layouts.split_enter && via_layouts.split_plus){
+    rgb_matrix_set_color(12, 0, 0, 0);
+    rgb_matrix_set_color(21, 0, 0, 0);
+    rgb_matrix_set_color(23, 0, 0, 0);
+  } else if (!(via_layouts.split_zero) && via_layouts.split_enter && via_layouts.split_plus){
+    rgb_matrix_set_color(12, 0, 0, 0);
+    rgb_matrix_set_color(21, 0, 0, 0);
+    rgb_matrix_set_color(22, 0, 0, 0);
+    rgb_matrix_set_color(24, 0, 0, 0);
+  } else if (via_layouts.split_zero && !(via_layouts.split_enter) && via_layouts.split_plus){
+    rgb_matrix_set_color(12, 0, 0, 0);
+    rgb_matrix_set_color(20, 0, 0, 0);
+    rgb_matrix_set_color(23, 0, 0, 0);
+    rgb_matrix_set_color(26, 0, 0, 0);
+  } else if (!(via_layouts.split_zero) && !(via_layouts.split_enter) && via_layouts.split_plus){
+    rgb_matrix_set_color(12, 0, 0, 0);
+    rgb_matrix_set_color(20, 0, 0, 0);
+    rgb_matrix_set_color(22, 0, 0, 0);
+    rgb_matrix_set_color(24, 0, 0, 0);
+    rgb_matrix_set_color(26, 0, 0, 0);
+  } else if (via_layouts.split_zero && via_layouts.split_enter && !(via_layouts.split_plus)){
+    rgb_matrix_set_color(11, 0, 0, 0);
+    rgb_matrix_set_color(16, 0, 0, 0);
+    rgb_matrix_set_color(21, 0, 0, 0);
+    rgb_matrix_set_color(23, 0, 0, 0);
+  } else if (!(via_layouts.split_zero) && via_layouts.split_enter && !(via_layouts.split_plus)) {
+    rgb_matrix_set_color(11, 0, 0, 0);
+    rgb_matrix_set_color(16, 0, 0, 0);
+    rgb_matrix_set_color(21, 0, 0, 0);
+    rgb_matrix_set_color(22, 0, 0, 0);
+    rgb_matrix_set_color(24, 0, 0, 0);
+  } else if (via_layouts.split_zero && !(via_layouts.split_enter) && !(via_layouts.split_plus)){
+    rgb_matrix_set_color(11, 0, 0, 0);
+    rgb_matrix_set_color(16, 0, 0, 0);
+    rgb_matrix_set_color(20, 0, 0, 0);
+    rgb_matrix_set_color(23, 0, 0, 0);
+    rgb_matrix_set_color(26, 0, 0, 0);
+  } else {
+    rgb_matrix_set_color(11, 0, 0, 0);
+    rgb_matrix_set_color(16, 0, 0, 0);
+    rgb_matrix_set_color(20, 0, 0, 0);
+    rgb_matrix_set_color(22, 0, 0, 0);
+    rgb_matrix_set_color(24, 0, 0, 0);
+    rgb_matrix_set_color(26, 0, 0, 0);
+  }
+}
+
+/* OLED stuff */
+#ifdef OLED_ENABLE
+bool oled_task_user(void) { 
+    render_anim();
+    
+    // WPM render
+    oled_set_cursor(0, 0);
+    uint8_t n = get_current_wpm();
+    char wpm_counter[6];
+    wpm_counter[5] = '\0';
+    wpm_counter[4] = '0' + n % 10;
+    wpm_counter[3] = '0' + (n /= 10) % 10;
+    wpm_counter[2] = '0' + n / 10 ;
+    wpm_counter[1] = '0';
+    wpm_counter[0] = '>';
+    oled_write_ln(wpm_counter, false);
```

Comment:
Could simplify this:
```c
    oled_write_P(PSTR(">0"), false);
    oled_write_ln(get_u8_str(get_current_wpm(), '0'), false);
```

It comes out to the same thing effectively, but there is already code in the core to handle this, basically.

Reply:
Ok, I'll change and test that ASAP.

Reply:
Done. I played around a bit with the code because it started rendering the first character of the default font (the one that looks like a sad face in https://joric.github.io/qle/). I couldn't fix it, so I just removed the first zero and centered the WPM counter.